UPGRADE-6.7.md

```diff
@@ -1,3 +1,130 @@
+# 6.7.1.0
+Currently, there are multiple order deliveries and multiple order transactions per order. If only one, the "primary", order delivery and order transaction is displayed and used in the administration, there is now an easy way in version 6.8 using the `primaryOrderDelivery` and `primaryOrderTransaction`. All existing orders will be updated with a migration so that they also have the primary values.
```

Comment:
title is missing for this section

Reply:
If there's no title, it's missing in the changelog file

Reply:
will open a new pr